init.go

```diff
@@ -217,6 +221,8 @@ func DefaultServicesConfigTestnet() Services {
 		HubDomain:          "https://hub-staging.btfs.io",
 		EscrowDomain:       "https://escrow-staging.btfs.io",
 		GuardDomain:        "https://guard-staging.btfs.io",
+		ExchangeDomain:     "https://exchange-dev.bt.co",
+		SolidityDomain:     "https://grpc.trongrid.io:50052",
```

Comment:
solidity domain is the same for testnet?? is there such a thing as a swap?

Reply:
I saw exchange using "grpc.shasta.trongrid.io" for testnet. I will update this.

Reply:
fixed